tests/pyfunc/test_scoring_server.py

```diff
@@ -545,7 +545,7 @@ def predict(self, context, model_input):
             model_uri="runs:/{}/model".format(run.info.run_id),
             data=json.dumps(df.to_dict(orient="split"), cls=NumpyEncoder),
             content_type=pyfunc_scoring_server.CONTENT_TYPE_JSON_SPLIT_ORIENTED,
-            extra_args=["--env-manager", "local"],
+            extra_args=["--env-manager", "None"],
```

Comment:
Flavor tests also contain `["--env-manager", "None"]`:

https://github.com/mlflow/mlflow/blob/b2687ccb5af1ed898a5bc2098008bb730b7d3119/tests/xgboost/test_xgboost_model_export.py#L36

I'll update them in a follow-up PR.